ui/imports/shared/popups/CommunityIntroDialog.qml

```diff
@@ -68,20 +50,16 @@ StatusDialog {
         }
     }
 
-    implicitWidth: Math.min(d.getMaxMinWidth(), d.maxWidth - d.getHorizontalPaddings())
-    implicitHeight: Math.min(columnContent.height + footer.height + header.height + d.getVerticalPaddings(), d.maxHeight)
-
-    StatusScrollView {
-        anchors.fill: parent
-        contentHeight: columnContent.height
-        contentWidth: columnContent.width
+    contentItem: StatusScrollView {
         padding: 0
+        implicitWidth: 300
```

Comment:
Here I'm wondering if it should be better to keep that : `Math.min(d.getMaxMinWidth(), d.maxWidth - d.getHorizontalPaddings())` since as a first shot, if the text is longer, I'm expecting to have a popup with a higher width (or the maximum one or the `introText.implicitWidth` if it is minor).

I.e. here I expect a poupup with a higher width:
![Screenshot 2023-02-17 at 13 42 23](https://user-images.githubusercontent.com/97019400/219656354-e743e233-a42f-404b-a9cb-1e0c3ebb16c7.png)

Reply:
Yup, I will increase the implicitWidth (minimum) here

Reply:
Fixed